toggle/Toggle.py

```diff
@@ -7,25 +7,24 @@
 gi.require_version('Cogl', '1.0')
 gi.require_version('Clutter', '1.0')
 
-from .Jog import Jog
-from .Network import Network, NetworkManager, ConnMan
-from .Settings import Settings
-from .Splash import Splash
-from .CubeTabs import CubeTabs
-from .FilamentGraph import FilamentGraph
-from .TemperatureGraph import TemperatureGraph
-from .Graph import Graph, GraphScale, GraphPlot
-from .Message import Message
-from .Event import Event, PushUpdate
-from .RestClient import RestClient
-from .WebSocksClient import WebSocksClient
-from .CascadingConfigParser import CascadingConfigParser
-from .Printer import Printer
-from .ModelLoader import ModelLoader
-from .VolumeStage import VolumeStage
-from .Plate import Plate
-from .Model import Model
-from .StyleLoader import StyleLoader
+from .core.CascadingConfigParser import CascadingConfigParser
+from .core.Network import Network, NetworkManager, ConnMan
```

Comment:
So are we supporting both connman and networkmanager now?

Reply:
Both have been supported from the beginning, but none of the have really worked well.